### DISCUSSION TITLE
Pixiv 图片反向代理 - Yuki 妙妙屋

### DISCUSSION THREAD

**Discussion by giscus** [xueelf/xueelf.github.io#46] [Announcements]
# Pixiv 图片反向代理 - Yuki 妙妙屋

众所周知，Pixiv 在大陆被墙，并且图片服务器域名 i.pximg.net 有盗链保护，只要 Referer 不是来自 Pixiv 的请求都会返回 403。那么，如何才能在大陆地区直接访问图片资源？ - Yuki - Yuki 妙妙屋

https://blog.yuki.sh/posts/599ec3ed8eda/

**Comment by Dituon** [comment]
可以使用域前置技术绕过防火墙封锁直接访问。

https://github.com/Dituon/pixiv-api-http/blob/main/HowToWork.md#%E7%BB%95%E8%BF%87%E9%98%B2%E7%81%AB%E5%A2%99

**Reply by xueelf** [reply]
好耶 ![](https://img.nga.178.com/attachments/mon_202008/16/-9lddQ5-eu5fK18T1kSc6-9r.jpg.thumb.jpg)

**Comment by Wudengnigui** [comment]
谢谢你

**Comment by SSSLiurong** [comment]
甚是感谢，帮助我解决了下载动图的问题

**Comment by yikuaiqianai** [comment]
一直以来谢谢你的图片代理链接！我是特地来访问的（一开始直接访问i.yuki.sh链接，一看是柚子，挺乐的。过了一段时间我又访问了yuki.sh发现了这个博客。），目的就是来感谢i yuki sh图片代理的开发者（拥有者）！自己使用的是pixez，平时软件自带的有点慢，好像还有高峰期，加载gif都挺慢的。但这个yuki挺快的，gif速度快了不少，但没有过多宣扬有这个链接代理，因为传播速度快非好事，也就默默使用这个好心人的图片代理，我也真心感谢这个链接的开发者！！谢谢你！Ciallo～(∠·ω< )⌒★

**Reply by xueelf** [reply]
诶嘿~ (◕ω＜)☆

**Comment by onlyfeather** [comment]
一开始是在你的博客里学会了建反代，后来服务器到期，就干脆直接用原主的东西了（莫名感觉有点丢脸），总之，很感谢开发者的无私奉献，这才是真的互联网精神啊（大拇指）

**Comment by Yorain00** [comment]
谢谢大佬的无私奉献！谢谢你！Ciallo～(∠·ω< )⌒★

**Reply by xueelf** [reply]
<img width="648" height="648" alt="Ciallo～(∠·ω< )⌒★" src="https://github.com/user-attachments/assets/e36b4ddd-c88f-4ca9-8e81-fc213c3f0e20" />

**Comment by God-2077** [comment]
感谢大佬的无私贡献

**Comment by hss2671** [comment]
感谢大佬的无私贡献

**Comment by 2393821643** [comment]
今天想用机器人随机几个图片，没想到停止服务了，非常感谢你的无私奉献。

**Comment by uuutt2023** [comment]
所以50刀能买多少黄游（XD）
